cloud/ec2.go

```diff
@@ -789,6 +789,18 @@ func (m *ec2Manager) TerminateInstance(ctx context.Context, h *host.Host, user,
 	if !IsEC2InstanceID(h.Id) {
 		return errors.Wrap(h.Terminate(ctx, user, fmt.Sprintf("detected invalid instance ID '%s'", h.Id)), "terminating instance in DB")
 	}
+
+	if h.NoExpiration {
```

Comment:
Just to double check my understanding: hosts that are currently considered unexpirable, once they sign up to follow a sleep schedule will no longer be considered unexpirable, right? And only hosts that have filed for an exception will be unexpirable?

Reply:
Unexpirable is a separate concept from the sleep schedule - it only means that Evergreen will never terminate the host unless they terminate it themselves (or make it expirable again). The sleep schedule only applies to unexpirable hosts.